docs/api/content-tracing.md

```diff
@@ -52,8 +52,6 @@ Once all child processes have acknowledged the `getCategories` request the
 ### `contentTracing.startRecording(options, callback)`
 
 * `options` Object
-  * `categoryFilter` String
-  * `traceOptions` String
```

Comment:
@MarshallOfSound How properly reflect the the docs the fact that the `options` Object can have two different formats?
First format is `{categoryFilter: "...", traceOptions: "..."}`, another one something like
```js
{
  "startup_duration": 4,
  "result_file": "/tmp/trace.json",
  "trace_config": {
    "included_categories": ["disabled-by-default-memory-infra"],
    "excluded_categories": ["*"],
    "memory_dump_config": {
      "triggers": [
        { "mode": "light", "periodic_interval_ms": 50 },
        { "mode": "detailed", "periodic_interval_ms": 1000 }
      ]
    }
  }
}
```

Reply:
It's a little 😬 that one of these formats uses `camelCase` and the other uses `snake_case`. Is it worth converting one of them?

Reply:
@nornagon the `{categoryFilter: "...", traceOptions: "..."}` variant is the only currently available, I don't want to break existing code.
The other one is a Chromium thing, and I don't want to overcomplicate it by adding format converting logic.